.pipelines/azure_pipeline_mergedbranches.yaml

```diff
@@ -27,12 +27,11 @@ jobs:
     name: Azure-Pipelines-CI-Test-EO
   steps:
   - task: ComponentGovernanceComponentDetection@0
-
+  ## commit will be in format 3.0.0-14-g833be407
```

Comment:
is this telemetry in AI matching with this image tag for both dev & prod images?

Reply:
the agent version will be in dev build format in AI telemetry like 3.1.0-12-commitId   while the agent version will be like 3.1.0

Reply:
what will be the version for the prod release in the telemetry?

Reply:
<img width="420" alt="image" src="https://user-images.githubusercontent.com/25573844/211433348-15911083-f4ce-4cc4-9195-e0a335d3a156.png">

Reply:
addressed 
<img width="658" alt="image" src="https://user-images.githubusercontent.com/25573844/212161964-d253235c-aa97-4b94-9d85-c5d0d20a7744.png">